### PR TITLE
Bump pylint to 2.15.2, update changelog

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
Too much manual changes to not make mistakes sadly :-(

### CODE REVIEW

**Review by DanielNoord — CHANGES_REQUESTED** [review]
There are two additional crashes/regressions reported today.

`astroid` `2.12.10`? 😅

**Comment by Pierre-Sassoulas** [comment]
It would be 2.12.9 I think ? I think we need to release, there's a lot of fix waiting in the pipeline and I botched 2.15.1 :(

I also need to fix the automated release tool because it's not as fast as I'd like right now but maybe when it's calmer.

**Comment by DanielNoord** [comment]
> It would be 2.12.9 I think ? I think we need to release, there's a lot of fix waiting in the pipeline and I botched 2.15.1 :(
> 
> I also need to fix the automated release tool because it's not as fast as I'd like right now but maybe when it's calmer.

I have a fix for the first crash regression almost ready. Just adding some tests.

I though of a solution for the other issue while in the shower this morning. Should be fairly straightforward as well.

If it isn't too much trouble I think it would be good to put them in as well.

**Comment by Pierre-Sassoulas** [comment]
All right then, let's release tonight.

**Comment by DanielNoord** [comment]
Both fixes have been pushed to `astroid`. One is already merged 😄

File: doc/whatsnew/2/2.15/index.rst [hunk 1]
```diff
@@ -30,10 +30,18 @@ Marc Byrne became a maintainer, welcome to the team !
...
+Release date: 2022-09-07
+
+- Fixed a case where custom plugins specified by command line could silently fail.
 
+  Specifically, if a plugin relies on the ``init-hook`` option changing ``sys.path`` before
+  it can be imported, this will now emit a ``bad-plugin-value`` message. Before this
+  change, it would silently fail to register the plugin for use, but would load
+  any configuration, which could have unintended effects.
+
+  Fixes part of #7264. (`#7264 <https://github.com/PyCQA/pylint/issues/7264>`_)
```
Comment (DanielNoord) [inline doc/whatsnew/2/2.15/index.rst:44]: Can we teach towncrier about whitespaces?

**Review by DanielNoord — CHANGES_REQUESTED** [review]

File: doc/whatsnew/2/2.15/index.rst (hunk 1 above)
Comment (Pierre-Sassoulas) [reply doc/whatsnew/2/2.15/index.rst:44]: What would you expect ?

Comment (DanielNoord) [reply doc/whatsnew/2/2.15/index.rst:44]: https://github.com/twisted/towncrier/issues/80

Setting `wrap` to `False` should work?

**Review by DanielNoord — APPROVED** [review]

**Pull request merged**